src/Pmi/Application/HpoApplication.php

```diff
@@ -96,7 +96,8 @@ protected function registerSecurity()
                 [['path' => '^/site($|\/)'], ['ROLE_USER', 'ROLE_AWARDEE']],
                 [['path' => '^/help($|\/)'], ['ROLE_USER', 'ROLE_ADMIN', 'ROLE_AWARDEE', 'ROLE_DV_ADMIN']],
                 [['path' => '^/settings($|\/)'], ['ROLE_USER', 'ROLE_ADMIN', 'ROLE_AWARDEE', 'ROLE_DV_ADMIN']],
-                [['path' => '^/.*$'], 'ROLE_USER']
+                [['path' => '^/biobank($|\/)'], 'ROLE_BIOBANK', 'ROLE_SCRIPPS'],
```

Comment:
Not sure, but should these two roles be in their own `[]`?

Reply:
Yes, I should be but somehow I missed it.